### DISCUSSION TITLE
Impossible connection to ttgo AP

### DISCUSSION THREAD

**Discussion by tk5ep** [dl9rdz/rdz_ttgo_sonde#313] [Q&A]
HI, asking for someone else who requested my help... :)
After flashing the latest version master_v0.9.1-B16-full, the access point address is displayed, but he cannot connect to it.
He tried with PDA and PC.
There is a X cross on the lower left on the screen that I never saw on my setup.

Any idea ?
![290920785_5727110127318755_2754004882937639518_n](https://user-images.githubusercontent.com/1655173/176392741-b93330eb-0cdf-42f7-b9c4-636f52ad2bdc.jpg)

**Comment by dl9rdz** [comment]
Nothing should have changed regarding the AP mode.
Maybe you can provide more details wrt what "cannot connect" means
- Does the client device "see" the access point?
- Can you connect to the access point (WiFi layer)? 
- If yes, does your client device get an IP address?
- If yes, then your problem is you cannot access http://192.168.4.1?
- Or did you try http://rdzsonde.local? This only works if your client device supprts mDNS discovery.

I assume that the X is from a recent pull request that I merged. It shows X if the T-Beam's internal GPS does not have a GPS fix (and a different symbol if it does).

**Reply by tk5ep** [reply]
Hi dl9rdz,
After dozens of confusing exchanges with the guy having problems, it seems that he can connect to the access point with his PDA. 
He doesn' have a wifi card on his PC.
Once connected, he can't access to the 192.168.4.1 address.

I hope it's the real situation. It looks a bit complicated to get an accurate answer from him. 

73 Patrick TK5EP

**Reply by tk5ep** [reply]
Any suggestion ?

**Reply by 73-de-LZ** [reply]
I saw this after OTA update DEV to DEV release few versions back, does not remember the version number. Than I deside to full flash the same version with esp tool via USB and the problem gone.

**Reply by tk5ep** [reply]
Thanks for the reply.
Everything works now. It was a misunderstanding of how the ttgo and smartphone have to work together !  :-(
I managed to explain that to the ham.

73, Patrick

**Reply by 73-de-LZ** [reply]
Very good, meanwhile I was able to reproduce the X mark before IP. 
If i flash devel20220531-B16-full.bin the X is present, when i flash devel20220426-B16-full.bin then OTA update via WiFi to devel20220531-B16 the X mark does not appear, but also there is no network symbol in the middle. In both cases no any problem with connectivity appear. I think the reason is in http://xxxxxxxx/edit.html?file=screens1.txt
@Scanner
last row is: 7,5=n
and if there is this row before it:
7,0=gV we get the unusual X mark on TTGO T3 v1.6.1
